fractal_tasks_core/lib_ngff.py

```diff
@@ -410,7 +429,15 @@ def load_NgffWellMeta(zarr_path: str) -> NgffWellMeta:
     Returns:
         A new `NgffWellMeta` object.
     """
-    zarr_group = zarr.open_group(zarr_path, mode="r")
+    try:
+        zarr_group = zarr.open_group(zarr_path, mode="r")
+    except GroupNotFoundError as e:
+        error_msg = (
+            f"\nOriginal error: {str(e)}.\n"
+            f"Zarr group not found at {zarr_path}."
```

Comment:
Suggestion: f"Could not load the requested well, because no Zarr group was found at {zarr_path}"

Reply:
Updated to
```python
f"Could not load attributes for the requested well, because no Zarr image was found at {zarr_path}"
```